Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -11492,6 +11492,29 @@ def data_error_handler(res):
     demisto.updateModuleHealth({'{data_type}Pulled'.format(data_type=data_type): data_size})
 
 
+def get_context_path(context, path):
+    """Get a context output ignoring the DT suffix.
```

Comment:
specify in the docs that this is meant to be used for outputs of executeCommand not on context gotten from the war room